staff/carlota-vicens/beers-app/scripts/logic.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
maybe it's easier to read if you use '' instead of "" for the use strict

Reply:
ho tindre en compte per la versio 2.0.0 @bcasasus